optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/common/ValidationListener.java

```diff
@@ -0,0 +1,8 @@
+package org.optaplanner.openshift.employeerostering.gwtui.client.common;
+
+import com.google.gwt.user.client.ui.HasValue;
+
+public interface ValidationListener {
```

Comment:
Doesn't errai/GWt have such an interface already?

Reply:
In Errai and GWT, typically it is the app developer's responsibility to call the validator, so there is no concept of validation events.